airbyte-integrations/connectors/destination-dev-null/src/main/kotlin/io/airbyte/integrations/destination/dev_null/DevNullWriter.kt

```diff
@@ -80,6 +81,22 @@ class LoggingStreamLoader(override val stream: DestinationStream, loggingConfig:
 
         return SimpleBatch(state = Batch.State.COMPLETE)
     }
+
```

Comment:
Does Devnull need to support file storage? If the interface defaults to unimplemented there's no need for any changes here.

Reply:
Done, discussed in office